snorkel/labeling/preprocess/core.py

```diff
@@ -11,13 +10,7 @@
 """
 
 
-class BasePreprocessor(BaseMapper):
-    """Base class for `Preprocessor` and `preprocessor`.
-
-    See `snorkel.map.core.Mapper` for details.
-    """
-
-    pass
+BasePreprocessor = BaseMapper
```

Comment:
Why did we drop the docstring for just this class and not the other tree trivial wrappers in this file? Also, as a browser of the code, if I see `BasePreprocessor`, `Preprocessor` and `LambdaPreprocessor`, all three of those sound like reasonable classes for me to use for a simple use case. And `BasePreprocessor` is currently only used as a type annotation. Can we clear it up somehow?

Reply:
mypy has trouble with this: some of the Mapper base classes use BaseMapper as the type of parameters and since this is a new class, it doesn't read BasePreprocessor as fulfilling that type correctly. As you saw, currently only used a type annotation, so documentation probably isn't needed.